client/src/components/Calendar.js

```diff
@@ -60,26 +61,8 @@ export default function Calendar({
     : totalDaysOfEachMonth;
 
   function handleModalOK() {
-    /*     if (dateSightCombos.some((combo) => combo.date === selectDate)) {
-      setDateSightCombos({
-        ...dateSightCombos,
-        date: [...dateSightCombos.date, sight.name],
-      });
-    } else { */
-    /*     setDateSightCombos([
-      ...dateSightCombos,
-      {
-        date: selectDate,
-        sight: sight.name,
-      },
-    ]); */
-    //}
-    let allCombos = dateSightCombos;
-    let newCombo = { date: selectDate, sight: sight.name };
-    allCombos.push(newCombo);
-    setDateSightCombos(...dateSightCombos, allCombos);
-
-    console.log(selectDate, sight.name, dateSightCombos);
+    const newCombo = { date: selectDate, sight: sight };
+    setDateSightCombos([...dateSightCombos, newCombo]);
```

Comment:
wasn't aware this works :) I would have pushed the value to App.js and made the state change there... but I guess if it works like that? :)

Reply:
works like a charm 😎